kernel/classes/ezorder.php

```diff
@@ -1353,6 +1353,7 @@ static function cleanupOrder( $orderID )
             $db = eZDB::instance();
             $db->begin();
             $db->query( "DELETE FROM ezorder where id='$orderID'" );
+            $db->query( "DELETE FROM ezorder_item WHERE order_id=$orderID" );
```

Comment:
Looking at line below, should this have been:
```suggestion
            $db->query( "DELETE FROM ezorder_item WHERE order_id=$orderNr" );
```

??